packages/bvaughn-architecture-demo/components/console/renderers/shared.module.css

```diff
@@ -2,26 +2,35 @@
 .ErrorRow,
 .TraceRow,
 .WarningRow {
-  padding: 0.5rem;
+  margin-top: -1px;
+  padding: 0.25rem 0.1rem;
   padding-left: 1.5rem;
+  border-top: 1px solid var(--background-color-contrast-1);
   border-bottom: 1px solid var(--background-color-contrast-1);
   position: relative;
 }
 .ErrorRow {
   background-color: var(--background-color-error);
   color: var(--color-error);
+  border-color: var(--border-color-error);
+  z-index: 5;
```

Comment:
I think this also caused a regression:
![Screen Shot 2022-09-16 at 2 56 12 PM](https://user-images.githubusercontent.com/29597/190711477-f7b6c186-9acc-42ef-b0f8-7c63b24153ca.png)